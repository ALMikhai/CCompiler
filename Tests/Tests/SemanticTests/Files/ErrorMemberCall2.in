struct a {
    int b;
} c1;

int f() {
    int a1 = c1.a;
}
