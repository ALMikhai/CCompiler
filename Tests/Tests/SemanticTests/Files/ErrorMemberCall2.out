(6:17): semantic error: 'a' has no member named 'a'
